packages/web-components/src/text-input/README.md

```diff
@@ -0,0 +1,134 @@
+| `disabled`      | public  | `boolean`             | `false`                 | Disables text input                                                                                                                                                                                                                        
+| `layout`        | public  | `TextInputLayout`     | `TextInputLayout.block` | Sets layout display property on text input                                                                                         |
+| `list`          | public  | `string`              |                         | Allows associating a `datalist` to an element by `id`                                                                              |
+| `maxlength`     | public  | `number`              |                         | The maximum number of characters a user can enter                                                                                  |
+| `minlength`     | public  | `number`              |                         | The minimum number of characters a user can enter                                                                                  |
+| `name`          | public  | `number`              |                         | The name of the control                                                                                                            |
+| `pattern`       | public  | `string`              |                         | A regular expression the text input's contents must match in order to be valid                                                     |
+| `placeholder`   | public  | `string`              |                         | An exemplar value to display in the text input field whenever it is empty                                                          |
+| `readonly`      | public  | `boolean`             | `false`                 | The text input should be submitted with the form but should not be editable                                                        |
+| `required`      | public  | `boolean`             | `false`                 | Sets the text input as required                                                                                                    |
+| `size`          | public  | `InputSize`           | `medium`                | Sets the size of the text input                                                                                                    |
+| `spellcheck`    | public  | `boolean`             | `false`                 | Controls whether or not to enable spell checking for the text input, or if the default spell checking configuration should be used |
+| `type`          | public  | `TextInputType`       | `TextInputType.text`    | Sets the size of the text input                                                                                                    |
+
```

Comment:
Do we want to document `value` as well?

Reply:
I'd also document `current-value` which will reflect and if not documented cause questions. I pointed @procload to an RFC on it's existence and why it's there. Probably good to include that and the reasoning behind it for awareness.

Reply:
Could you point me to that RFC?